### DISCUSSION TITLE
1B32EDCD-7C9E-4694-B33A-DED45FD68369

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#41] [General]
# G-Man

G-Man from the Half-Life series produced by Valve. Later updated to include the characters of Team Fortress 2, also made by Valve. The models is originally from the Modeler's Resource and converted to CMOD by Anthony_B_Russo10.

https://celestia.mobi/addon?item=1B32EDCD-7C9E-4694-B33A-DED45FD68369&lang=en

**Comment by Dirgham2015** [comment]
Wow a man

**Comment by TheLostProbe** [comment]
Gordon Freeman in the flesh. Or, rather, in the OpenGL space simulator

**Comment by LangLangFCS** [comment]
It's funny
